test_data/makedata_suites/610_analyzers.js

```diff
@@ -0,0 +1,255 @@
+/* global print, semver, progress, createSafe, db, PWD*/
+/*jslint maxlen: 130 */
+
+(function () {
+  const a = require("@arangodb/analyzers");
```

Comment:
General comment about that test itself: 

I only do see positive tests, where we do not expect any failures. 
Do we also want to test things here which are expected to fail?

e.g. - Wrong binary path given, wrong list of features, wrong binary format etc.

Reply:
In which environments can this test be run? 
Is this suite also being executed against Oasis? 
If so, how can we upload binaries there?

Reply:
Additionally, we do have such tests quite close to to the defined ones here in `tests/js/common/aql/aql-view-arangosearch-feature.js` (Main repository):
- `testCustomClassificationAnalyzer`
- `testCustomNearestNeighborsAnalyzer`

In general: What is the benefit of having them here as well in comparison to the ones in our main repository?

Reply:
at the moment they break dc2dc ;)